packages/ai-bot/lib/matrix.ts

```diff
@@ -67,42 +67,32 @@ export async function sendMessage(
   );
 }
 
+export interface FunctionToolCall {
+  name: 'searchCard' | 'patchCard';
+  arguments: any;
```

Comment:
i would imagine that the arguments for the tool call might be specific to the tool being called. Perhaps we have a type for each function tool, and `FunctionToolCall` is the intersection of all the functino tool interfaces.

like:
```ts
interface SearchToolCall {
  type: 'search';
  arguments: Query
}

interface PatchToolCall {
  type: 'patch;
  arguments: PatchData; // this is a real type we should probably be using it
}

type FunctionToolCall = SearchToolCall | PatchToolCall;
```

we should really strive to not use `any`, if we do let's try to justify why we think the type is any, Eventually I imagine that these command specific types will probably be separated into skill cards or something of the like.

Reply:
As in the previous PR, We will keep all this code in the command service for now until the command field api is introduced